app/gui/view/graph-editor/src/builtin/visualization/java_script/table.js

```diff
@@ -11,7 +11,7 @@ loadStyleFromString(scrollbarStyle)
 // ===========================
 
 class TableVisualization extends Visualization {
-    static inputType = 'Any'
+    static inputType = 'Standard.Table.Data.Table.Table'
```

Comment:
I tried building the IDE to verify this, but after wasting a lot of time I'm still having some troubles using the new build script on Windows (I will try figure this out in due course).

In the meantime, I have a hunch (the one I wanted to verify before posting, but got impatient): if we specify the inputType of visualization to be `Standard.Table.Data.Table.Table`, will the Table visualization even **show at all** for Database Tables whose type is `Standard.Database.Data.Table.Table`? I think it is important to not break the Table visualization on the Database module and would consider breaking it as a regression.

I'm not sure if this PR causes it (as I was unable to verify empirically), but looking at the code I'm afraid it may.

btw. cc @jdunkerley I think this is another argument for why we should migrate towards a single _facade_ Table type that will only then delegate to the proper backend (be it DB backend or in-memory) under the hood. Having one Table type instead of 2 (or more) will be much better. Will probably be worth doing as part of the SQL IR improvements.

For now though, it would be good to ensure if we can keep the Table visualization on Database Tables too. Can we use sum types in `inputType`?

Reply:
While this is only a temporary solution until such time as we have the method calling `default_visualization`, @radeusgd is correct we need this on a few more types. Could we make the inputType an array of types instead of a single value?

For now, Table viz should be the default for Vector, Array, In Memory Column and In Memory Table.
And SQL should be the default for Database Column and Database Table.

We are **very** likely to increase this list over the next few days (I want to name Map and JS Object for example default to the Table) so this mechanism is critical assuming that #5195 will not be completed for a while.

Reply:
@radeusgd - I don't think the facade type has anything to do here. 

And at least so far attempts to use such a type within Enso have been annoying. I tried this for the database connection type with a facade calling onto a backing true connection but the approach was very unpleasant and didn't work well. Either way, a discussion for a different space.

Reply:
There can be any number of types, but there is one limitation: the matching is done on the name as a string. The only special handling is done for `Any`. So I can add that back in to ensure this stays usable for all types. And I can add more types for which it makes sense for this to be the default.

So the list of types to add is

* `Standard.Database.Data.Table.Table`
* `Vector`
* `Array`
* `In Memory Column`
*  `In Memory Table`

Any other ones that come to mind? @jdunkerley @radeusgd

Reply:
yes please. For the first pass:

Table:
- `Standard.Base.Data.Vector.Vector`
- `Standard.Base.Data.Array.Array`
- `Standard.Table.Data.Table.Table`
- `Standard.Table.Data.Column.Column`

and lets make SQL viz the default for:
- `Standard.Database.Data.Table.Table`
- `Standard.Database.Data.Column.Column`

We can then hopefully easily add to this as we add need.

Reply:
@MichaelMauderer / @farmaazon  is #5195 planned for the release?

Reply:
I'd just like to ensure that while for
- `Standard.Database.Data.Table.Table`
- `Standard.Database.Data.Column.Column`
we have the SQL visualization as the default, the `Table` visualization should still be available to choose for these types.

Reply:
yes the Table viz should definitely be available for it.

Reply:
> 



> Table:
> 
>     * `Standard.Base.Data.Vector.Vector`
> 
>     * `Standard.Base.Data.Array.Array`
> 
>     * `Standard.Table.Data.Table.Table`
> 
>     * `Standard.Table.Data.Column.Column`
All added. 


> and lets make SQL viz the default for:
> 
>     * `Standard.Database.Data.Table.Table`
> 
>     * `Standard.Database.Data.Column.Column`

Already seem to be defined as the input for SQL vis, so they should have it as default now.

Reply:
> @MichaelMauderer / @farmaazon is #5195 planned for the release?

I don't think it is. @farmaazon ?

Reply:
> I'd just like to ensure that while for
> 
>     * `Standard.Database.Data.Table.Table`
> 
>     * `Standard.Database.Data.Column.Column`
>       we have the SQL visualization as the default, the `Table` visualization should still be available to choose for these types.

I've re-added “Any” to Table Vis can be chosen for every node. This might be the best compromise here until we get the proper system.